src/components/PrivacyDisclaimer.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Button } from 'antd';
+
+const Footer = styled.div`
+  width: 100vw;
+  overflow: hidden;
+  padding: 12px 40px;
+  display: flex;
+  border-top: 1px solid #efefef;
+  background-color: #fff;
+  align-items: center;
+  p {
+    flex: 1;
+    margin-right: 12px;
+    margin-bottom: 0px;
+  }
+  button.dismiss {
+    margin-right: 0px;
+  }
+  button {
+    margin-right: 12px;
+  }
+`;
+
+export const PRIVACY_ACCEPT = 'acceptPrivacy';
+
+class PrivacyDisclaimer extends React.Component {
+  componentDidMount() {
+    if (localStorage.getItem(PRIVACY_ACCEPT) === null) {
+      setTimeout(() => {
```

Comment:
Hmm must website once you choose to accept, it would stop showing a disclaimer (i.e New York Times, Guardian, etc.) But theyre also dealing with more trackers and cookies... again no policy so up for debate :P

Reply:
Using mixpanel puts us in the same category, I think we should use the same behavior pattern for two reasons. 1. People are used to dismiss cookie warnings and 2. Behind the scene "magic" behavior is confusing.

Reply:
Okay, I'll show it until the user dismisses it then :)